lists/blocklist.yaml

```diff
@@ -44,3 +44,63 @@
   - url: staking.xyz
   - url: gemrodeo.xyz
   - url: jupdrop66.com
+  - url: shdw-get.netlify.app
+  - url: findgem.xyz
+  - url: mevsol.com
+  - url: jitovoucher.org
+  - url: circleonsol.com
+  - url: ftxreborn.com
+  - url: collab-award.io
+  - url: claimray.net
+  - url: jup.pro
+  - url: moonroll.io
+  - url: dropbonk.space
+  - url: guac-drop.netlify.app
+  - url: jupjup2.com
+  - url: receiveusdc.com
+  - url: tensor-samurai.io
+  - url: tensorgift.pw
+  - url: bonkgifts.com
+  - url: solana.ag
+  - url: claimbonk.online
+  - url: jup-drop.space
+  - url: npoint.io
+  - url: pythdrop.site
+  - url: phantomupdate.com
+  - url: solusdc.fun
+  - url: boner-get.netlify.app
+  - url: bork-airdrop-.xyz
+  - url: elote-drop.netlify.app
+  - url: gotbit-claim.netlify.app
+  - url: jup-airdrop.xyz
+  - url: koi-earn.netlify.app
+  - url: claimlido.xyz
+  - url: mylido.xyz
+  - url: getlido.site
+  - url: nos-get.netlify.app
+  - url: omni-claim.netlify.app
+  - url: orca-airdrop.xyz
+  - url: pants-claim.netlify.app
+  - url: popcat-get.netlify.app
+  - url: whales-get.netlify.app
+  - url: dofwifhat.netflify.app
+  - url: raydi.io
+  - url: get-bonk.netlify.app
+  - url: bonkget.com
+  - url: sol-drift.com
+  - url: jupclaim.net
+  - url: hivemapper.online
+  - url: staratlas.bio
+  - url: jupiter-drop.netlify.app
+  - url: meme-draw.io
+  - url: solanbox.com
+  - url: soldrop.online
+  - url: pyth.bio
+  - url: dropray.site/
+  - url: rollbitking.com
+  - url: sol2024.info
+  - url: nftarea.net/
+  - url: wenclaim.net
```

Comment:
Remove "/" from urls like "nftarea.net/" and others

Reply:
removed, thank you